apps/ios/Shared/Views/Chat/ChatView.swift

```diff
@@ -596,10 +671,10 @@ struct ChatView: View {
             }
         }
 
-        private func menu(live: Bool) -> [UIMenuElement] {
+        private func menu(_ ci: ChatItem, _ range: ClosedRange<Int>?, live: Bool) -> [UIMenuElement] {
```

Comment:
ci is passed to menu as it may be another item from merged group and not the main item